excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ListFieldTypeParser.java

```diff
@@ -28,18 +29,20 @@
     private static final String FIXED_LIST_TYPE = "FixedList";
     private static final String FIXED_RADIO_LIST_TYPE = "FixedRadioList";
     private static final String MULTI_LIST_TYPE = "MultiSelectList";
+    private static final String DYNAMIC_LIST_TYPE = "DynamicList";
     public static final String NO_BASE_TYPE_FOUND = "No base type found for: ";
     private final ParseContext parseContext;
     private final FieldTypeEntity fixedListBaseType;
     private final FieldTypeEntity fixedRadioListBaseType;
     private final FieldTypeEntity multiListBaseType;
+    private final FieldTypeEntity dynamicListBaseType;
 
     public ListFieldTypeParser(ParseContext parseContext) {
         this.parseContext = parseContext;
         fixedListBaseType = parseContext.getBaseType(FIXED_LIST_TYPE).orElseThrow(() -> new InvalidImportException(NO_BASE_TYPE_FOUND + FIXED_LIST_TYPE));
         fixedRadioListBaseType = parseContext.getBaseType(FIXED_RADIO_LIST_TYPE).orElseThrow(() -> new InvalidImportException(NO_BASE_TYPE_FOUND + FIXED_RADIO_LIST_TYPE));
         multiListBaseType = parseContext.getBaseType(MULTI_LIST_TYPE).orElseThrow(() -> new InvalidImportException(NO_BASE_TYPE_FOUND + MULTI_LIST_TYPE));
-
+        dynamicListBaseType = parseContext.getBaseType(DYNAMIC_LIST_TYPE).orElseThrow(() -> new InvalidImportException(NO_BASE_TYPE_FOUND + DYNAMIC_LIST_TYPE));
```

Comment:
the ticket says: 
```If a list has been been defined using 'FieldTypeParameter' when it is 'DynamicList' it configured list should be ignored```
given this requirement, can we just remove this line here? Because we have to ignore this info anyway

Reply:
perhaps removing that line will fix RDM-4634?